src/core_write.cpp

```diff
@@ -170,9 +170,9 @@ void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fInclud
 
         CAssetTransfer data;
         if (GetTransferAsset(scriptPubKey, data)) {
-            std:string assetId = data.AssetId;
```

Comment:
This one surprised me - a single colon?  I wonder what the compiler was thinking - lol.